locust/contrib/postgres.py

```diff
@@ -0,0 +1,49 @@
+from locust import TaskSet, User, events
+
+import time
+
+import psycopg
+
+
+def create_conn(conn_string):
+    return psycopg.connect(conn_string)
+
+
+def execute_query(conn_string, query):
+    db_conn = create_conn(conn_string)
```

Comment:
Does this create a new connection on each query? That sounds very wasteful. Probably you wanna do that only when the client object is created.